adm/src/blockstore.rs

```diff
@@ -0,0 +1,122 @@
+use protobuf;
+use protobuf::Message;
+use sawtooth_sdk::messages::block::{Block, BlockHeader};
+
+use database::database::DatabaseError;
+use database::lmdb::LmdbDatabase;
+
+// TODO: Wrap block protobuf in BlockWrapper
```

Comment:
Remove the todo.

Reply:
Needed to implement this in order for the show command to work.